translation-cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
@ptewson-sfdo Can you move this to a `localization` folder to match the other repos?

Reply:
Also, you're probably better off using the same file from the Contacts & Orgs repo.